time set_image_flag_time
---
bool set_success